packages/schemas/tables/logto_settings.sql

```diff
@@ -0,0 +1,5 @@
+create table _logto_settings (
```

Comment:
configs?

Reply:
Per discussion, use `_logto_settings` since we already have a table with the name "settings".